hpx/plugins/parcel/message_buffer.hpp

```diff
@@ -60,7 +60,27 @@ namespace hpx { namespace plugins { namespace parcel { namespace detail
         void operator()(parcelset::parcelport* pp)
         {
             if (!messages_.empty())
+            {
+                if (0 == threads::get_self_ptr())
```

Comment:
Am 04.06.2016 1:33 nachm. schrieb "Hartmut Kaiser" <notifications@github.com
>:
>
> In hpx/plugins/parcel/message_buffer.hpp:
>
> > @@ -60,7 +60,27 @@ namespace hpx { namespace plugins { namespace parcel
{ namespace detail
> >          void operator()(parcelset::parcelport* pp)
> >          {
> >              if (!messages_.empty())
> > +            {
> > +                if (0 == threads::get_self_ptr())
>
> Hmmm, going through the parcel handler again would force the created
(coalesced) parcels to go through coalescing again - not sure if we can
prevent this.

Good point, haven't thought about that...

>
> Also, in case of migration, the destination will resolve this just fine.

Right, overlooked the additional check inside of schedule_action.

>
> In some sense, the parcel-coalescing is a parcel-handler on its own...

Good point. It's probably fine then.

>
> —
> You are receiving this because you commented.
>
> Reply to this email directly, view it on GitHub, or mute the thread.